teams.py

```diff
@@ -40,3 +40,15 @@ def __getitem__(self, item):
 
     def __len__(self):
         return len(self.team)
+
+    def find_stronges_warrior(self):
+        pass
+
```

Comment:
Jak to oponent. Przecież szuka wewnątrz swoich członków

Reply:
?

Reply:
metoda "find_weakest_opponent" szuka nasłabszego gracza we własnej drużynie. To nie jest oponent.